Makefile

```diff
@@ -68,7 +68,9 @@ run-flask-with-docker: ## Run flask in Docker container
 
 .PHONY: test
 test: test-requirements ## Run tests (used by Concourse)
-	./scripts/run_tests.sh
+	flake8 .
+	isort --check-only ./app ./tests
+	PYTHONPATH=. pytest
```

Comment:
Is the loss of `-vv` here intentional?

Reply:
Yeah, I'm letting the scope creep a bit, sorry.

We don't use `-vv` uniformly in our apps. By default, I think we should use the concise output; if someone wants the verbose output, or other bespoke options, they should run `pytest` manually with their preferred options.

I also don't mind putting it back if you'd prefer to leave it as-is 👍.